modules/core/shared/src/main/scala/gem/config/DynamicConfig.scala

```diff
@@ -291,4 +292,7 @@ object DynamicConfig {
     )
   }
 
+  implicit val EqDynamicConfig: Eq[DynamicConfig] =
+    Eq.fromUniversalEquals // TODO: ensure this is ok
```

Comment:
Good question

Reply:
I think it's fine because it's an ADT that eventually goes down to ground types, but it's bad practice to assume this because it can become wrong in the future. I think we can come back later to review model equality. Opened #541